verification/dependencies/net/netip/netip.gobra

```diff
@@ -44,9 +49,19 @@ pure
 decreases _
 func (ip Addr) IsValid() (res bool)
 
+decreases _
+func (ip Addr) Is4() bool
+
 decreases _
 func (ip Addr) Is6() bool
 
 ensures ip.IsValid() ==> sl.Bytes(b, 0, len(b))
 decreases _
 func (ip Addr) AsSlice() (b []byte)
+
+pure
```

Comment:
Why are we making this function pure? Even though I don't see a reason why it should be side-effectful, I would still rather introduce a ghost spec function `IsValidSpec` and specify `a.IsValid()` to return the same value as `a.IsValidSpec()` (i.e., if we actually have to talk about it in specs)

Reply:
I made this function pure since I wanted to use it in the specs (see for example `func New` in `conn.go` line 104). I also checked the implementation to see that it actually is side-effect free.

But sure, I can also introduce a ghost spec function for this purpose. 
Should I do the same for `func (ip Addr) IsValid() (res bool)`? I also made this pure for the verification of the `slayers` package.

Reply:
Done.